src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectExecContainerResponseHandler.groovy

```diff
@@ -0,0 +1,21 @@
+package com.bmuschko.gradle.docker.response.container
+
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+
+/**
+ * Logs inspect exec container result.
+ *
+ * @see com.bmuschko.gradle.docker.tasks.container.DockerInspectExecContainer
+ */
+class InspectExecContainerResponseHandler {
```

Comment:
Just getting back around to this now. I really think we should remove this in favor of using an `onNext` block and if none is passed than create one. Within a few releases I'd like to remove any use of the "response handler" objects/concept as I believe the reactive-streams impl is a superior solution. Keeping it in, and/or adding to it, may give the wrong impression that we want to keep this around.

Aside from that: everything else looks good.